src/Apps/Sell/Utils/acceptableConditionsForSubmission.tsx

```diff
@@ -0,0 +1,58 @@
+import { compact } from "lodash"
+import { ArtworkConditionEnumType } from "__generated__/useUpdateArtworkMutation.graphql"
+
+type AcceptableValuesMapKey = Exclude<
+  ArtworkConditionEnumType,
+  "%future added value"
+>
+
+export type AcceptableConditionValue = AcceptableValuesMapKey
+
+// By having this ACCEPTABLE_VALUES_MAP structure, we are forced to update this list
+// whenever ArtworkConditionEnumType changes, because yarn tsc will fail
+
+export const ACCEPTABLE_CONDITION_VALUES_MAP: Record<
+  AcceptableValuesMapKey,
+  AcceptableConditionValue
+> = {
+  EXCELLENT: "EXCELLENT",
+  FAIR: "FAIR",
+  GOOD: "GOOD",
+  VERY_GOOD: "VERY_GOOD",
+}
+
+export const conditionOptions = [
+  { text: "Select Condition", value: "" },
+  { text: "Excellent Condition", value: "Excellent" },
+  { text: "Fair Condition", value: "Fair" },
+  { text: "Good Condition", value: "Good" },
+  { text: "Very Good Condition", value: "Very Good" },
+]
+
+export const formatConditionValueForSubmission = (conditionValue: string) => {
+  return conditionValue
+    .split(/[^A-Za-z]/)
+    .reduce((accumulator, current) => {
+      if (current && current.trim()) {
+        accumulator.push(current.toUpperCase())
+      }
+      return accumulator
+    }, [] as string[])
+    .join("_") as AcceptableConditionValue
+}
+
+export const acceptableConditionsForSubmission = () => {
```

Comment:
Hmm, what is this function doing and why can't we just use `conditionOptions` or a hardcoded array? This function looks very complicated and it does not have any input and would always return the same array anyway, right?

Reply:
I thought since we have a `ArtworkConditionEnumType` defined, we could make sure when it changes, we will be forced to update the array we use for the `<Select />` component. If we hardcode the values, there is a chance we miss this change. 
I followed the implementation from `acceptableCategoriesForSubmission.ts`
If it sounds like too much, we totally can use `conditionOptios` or a hardcoded array

Reply:
I've checked it, and I'm not quite sure if it works. When I change a value in `conditionOptions`, I don't see a TypeScript error.

Could we alternatively just use a typed array in `ConditionRoute.tsx` and remove `acceptableConditionsForSubmission.tsx` and `const conditionOptions = useMemo(acceptableConditionsForSubmission, [])`? 

The following should give us a TypeScript error whenever the enum in the schema changes:

```typescript
// This type is here to make sure the condition options match the allowed values in the schema. We have to update the conditionOptions array whenever the ArtworkConditionEnumType changes.
type ConditionOption = {
  text: string
  value: Exclude<ArtworkConditionEnumType, "%future added value"> | ""
}

export const conditionOptions: ConditionOption[] = [
  { text: "Select Condition", value: "" },
  { text: "Excellent Condition", value: "EXCELLENT" },
  { text: "Fair Condition", value: "FAIR" },
  { text: "Good Condition", value: "GOOD" },
  { text: "Very Good Condition", value: "VERY_GOOD" },
]
```